content/publication_resources/bibtex/lahti.bib

```diff
@@ -12,10 +12,61 @@ @Article{
   annote =	 {}
 }
 
-    
 
 
-    
+@Article{McDonald2023,
+  author =	 {Daniel McDonald, Yueyu Jiang, Metin Balaban, Kalen Cantrell,
+            Qiyun Zhu, Antonio Gonzalez, James T. Morton, Giorgia Nicolaou, 
+            Donovan H. Parks, Søren M. Karst, Mads Albertsen, Philip Hugenholtz, 
+                Todd DeSantis, Se Jin Song, Andrew Bartko, Aki S. Havulinna,
+              Pekka Jousilahti, Susan Cheng, Michael Inouye, Teemu Niiranen, 
+          Mohit Jain, Veikko Salomaa, Leo Lahti, Siavash Mirarab and Rob Knight},
+  title =	 {Greengenes2 unifies microbial data in a single reference tree},
+  journal =	 {Nature Biotechnology},
+  year =	 2023,
+  doi =		 {10.1038/s41587-023-01845-1},
+  keywords =	 {bioscience}
```

Comment:
In bibtex you must use either "Last, First" or "First Last" (without comma); and the names _must_ be separater with "and". Otherwise the interpretation of last/first names will go wrong in the rendering.

-> to fix for all refs